src/domains/gps-app/api/payments-service/v1/rest/external/_base_policy.xml

```diff
@@ -0,0 +1,30 @@
+<policies>
+    <inbound>
+        <set-variable name="requestPath" value="@(context.Request.Url.Path)" />
+        <set-variable name="sub" value="@(context.Subscription.Name)" />
+        <set-variable name="subid" value="@(context.Subscription.Id)" />
+        <set-variable name="subkey" value="@(context.Subscription.Key)" />
+        <set-variable name="fiscalCodeParam" value="@(context.Request.MatchedParameters["organizationfiscalcode"])">
+        <set-variable name="organizationFiscalCode" value="@{
+            return context.Variables.ContainsKey("User.Id") ? context.User.Id : "Unknown" ;
+        }" />
+        <base />
+        <choose>
+            <when condition="@(context.User == null || !fiscalCodeParam.Equals(organizationFiscalCode))">
```

Comment:
remove `context.User == null` it's useless

Reply:
👍